packages/cdk/lib/util/index.ts

```diff
@@ -124,9 +125,13 @@ export const renderPythonLambda = (
   codePath: string,
   environment: Record<string, string>
 ): PythonFunction => {
-  return new PythonFunction(scope, id, {
+  const account: string = process.env.CDK_DEFAULT_ACCOUNT!;
+  const region: string = process.env.CDK_DEFAULT_REGION!;
+  const bucketName = `${APP_NAME}-${account}-${region}`;
```

Comment:
I think we are already doing this in other places. Can we re-use?

Reply:
Although we are doing the same thing in the app stack, our code from go calls the cdk stacks separately (current change is in core stack)